test/clib/Makefile

```diff
@@ -32,11 +32,21 @@ libnispor.so: $(DEBUG_NISPOR_SO_FILE)
 nispor_test: nispor_test.c nispor.h libnispor.so
 	$(CC) $(CFLAGS) $(LDFLAGS) -o nispor_test nispor_test.c $(LIBS)
 
+run: nispor_test
+	sudo \
+	RUST_BACKTRACE=1 \
+	LD_LIBRARY_PATH=$(TMPDIR) \
+		./nispor_test
+	rm -rf $(TMPDIR)
+
+
 check: nispor_test
+	sudo \
+	RUST_BACKTRACE=1 \
 	LD_LIBRARY_PATH=$(TMPDIR) \
 		valgrind --trace-children=yes --leak-check=full \
 		--error-exitcode=1 \
-		./nispor_test 1>/dev/null
+		./nispor_test
```

Comment:
This is unrelated to this PR.

Reply:
But without the output you don't see what is wrong not all the output is at stderr when there is an error.